### ISSUE TITLE
When ShowTitleBar="False",Canvas can't fire MouseDown (using the left button) and others

### ISSUE THREAD

**Issue by sunfly2015** [MahApps/MahApps.Metro#1742]
When ShowTitleBar = "False", Canvas can not properly trigger MouseDown (using the left button), PreviewMouseDown (using the left button), MouseLeftButtonDown, PreviewMouseLeftButtonDown, completely without triggering MouseUp (using the left button), PreviewMouseUp (using the left button), MouseLeftButtonUp, PreviewMouseLeftButtonUp.

These bugs in the eight months ago, someone submitted.
After upgrading Mahapps.Metro to version 1.0 does not solve These bugs.

```xaml
<UserControl x:Name="userControl" x:Class="ControlLibrary.GlowButton"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
             mc:Ignorable="d" 
             d:DesignHeight="300" d:DesignWidth="300" >
	<UserControl.Resources>
		<Storyboard x:Key="OnMouseEnter1">
			<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.BlurRadius)" Storyboard.TargetName="userControl">
				<EasingDoubleKeyFrame KeyTime="0:0:0.2" Value="12"/>
			</DoubleAnimationUsingKeyFrames>
		</Storyboard>
		<Storyboard x:Key="OnPreviewMouseDown1">
			<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.BlurRadius)" Storyboard.TargetName="userControl">
				<EasingDoubleKeyFrame KeyTime="0:0:0.2" Value="12"/>
			</DoubleAnimationUsingKeyFrames>
			<ColorAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.Color)" Storyboard.TargetName="userControl">
				<EasingColorKeyFrame KeyTime="0:0:0.2" Value="Black"/>
			</ColorAnimationUsingKeyFrames>
		</Storyboard>
		<Storyboard x:Key="OnPreviewMouseUp1">
			<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.BlurRadius)" Storyboard.TargetName="userControl">
				<EasingDoubleKeyFrame KeyTime="0:0:0.2" Value="12"/>
			</DoubleAnimationUsingKeyFrames>
			<ColorAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.Color)" Storyboard.TargetName="userControl">
				<EasingColorKeyFrame KeyTime="0:0:0.2" Value="White"/>
			</ColorAnimationUsingKeyFrames>
		</Storyboard>
		<Storyboard x:Key="OnMouseLeave1">
			<ColorAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.Color)" Storyboard.TargetName="userControl">
				<EasingColorKeyFrame KeyTime="0:0:0.2" Value="White"/>
			</ColorAnimationUsingKeyFrames>
			<DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Effect).(DropShadowEffect.BlurRadius)" Storyboard.TargetName="userControl">
				<EasingDoubleKeyFrame KeyTime="0:0:0.2" Value="2"/>
			</DoubleAnimationUsingKeyFrames>
		</Storyboard>
    </UserControl.Resources>
	<UserControl.Triggers>
		<EventTrigger RoutedEvent="UIElement.MouseEnter">
			<BeginStoryboard Storyboard="{StaticResource OnMouseEnter1}"/>
		</EventTrigger>
		<EventTrigger RoutedEvent="UIElement.MouseLeave">
			<BeginStoryboard x:Name="OnMouseLeave1_BeginStoryboard" Storyboard="{StaticResource OnMouseLeave1}"/>
		</EventTrigger>
		<EventTrigger RoutedEvent="UIElement.PreviewMouseDown" SourceName="userControl">
			<BeginStoryboard x:Name="OnPreviewMouseDown1_BeginStoryboard" Storyboard="{StaticResource OnPreviewMouseDown1}"/>
		</EventTrigger>
		<EventTrigger RoutedEvent="UIElement.PreviewMouseUp" SourceName="userControl">
			<BeginStoryboard x:Name="OnPreviewMouseUp1_BeginStoryboard" Storyboard="{StaticResource OnPreviewMouseUp1}"/>
		</EventTrigger>
	</UserControl.Triggers>
	<UserControl.Effect>
		<DropShadowEffect BlurRadius="0" Direction="0" ShadowDepth="0" Color="White"/>
	</UserControl.Effect>
    <Grid x:Name="grid">
        <Rectangle x:Name="_Rect" Fill="{Binding Foreground, RelativeSource={RelativeSource AncestorType={x:Type UserControl}}}">
            <Rectangle.OpacityMask>
                <VisualBrush Stretch="Fill" Visual="{Binding Visual, RelativeSource={RelativeSource AncestorType={x:Type UserControl}}}" />
            </Rectangle.OpacityMask>
        </Rectangle>
    </Grid>
</UserControl>
```

**Comment by punker76** [comment]
@sunfly2015 i don't see any `Canvas` in your sample.

**Comment by sunfly2015** [comment]
VisualBrush.Visual binded to a Canvas.
![image3](https://cloud.githubusercontent.com/assets/10437586/5664021/cbec61ae-9787-11e4-9d47-dadb99ee7e14.jpg)
Look,The effect of the blue part of the code is not normal.

I can't speak English,So I input these too slowly.

**Comment by sunfly2015** [comment]
![image4](https://cloud.githubusercontent.com/assets/10437586/5664160/1452d6a2-9789-11e4-99d6-eaef0e985455.jpg)
This is the instance above the user control.The Visual property binded to a Canvas.
![image5](https://cloud.githubusercontent.com/assets/10437586/5664230/b7a2ba3e-9789-11e4-83c3-27b0b45c2aea.jpg)
I must go to sleep,hope tomorrow is a good day!

**Comment by punker76** [comment]
should be fixed in v1.2.3
